src/QMCWaveFunctions/tests/test_kspace_jastrow.cpp

```diff
@@ -0,0 +1,126 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
```

Comment:
Please put `Copyright (c) 2019 QMCPACK developers.` and update the following with your name and affiliation.

Reply:
updated